test/Thermal/thermal.jl

```diff
@@ -32,7 +32,7 @@ D = Differential(t)
     # Check if Relative temperature sensor reads the temperature of heat capacitor
     # when connected to a thermal conductor and a fixed temperature source
     @test sol.retcode == Success
-    @test sol[reltem_sensor.T] + sol[tem_src.port.T] == sol[mass1.T] + sol[th_conductor.dT]
+    @test sol[reltem_sensor.T] + sol[tem_src.port.T] ≈ sol[mass1.T] + sol[th_conductor.dT]
```

Comment:
I am not sure about these ones. Shouldn't derived variables be exact?

Reply:
DAECompiler is off by a few ulps, I'm guessing because state tearing chose different states than MTK did, causing the reconstruction of these values to perform floating-point operations in a different order.  I haven't looked deeply into it, I just saw that the answer was off by a few ulps so I changed it to `≈`.